### DISCUSSION TITLE
Logging exporter to a file [log level INFO]

### DISCUSSION THREAD

**Discussion by Nighttraveler** [open-telemetry/opentelemetry-collector#5165] [Q&A]
Hi guys is there a way to configure the output of the logging exporter to a file? We want to get the INFO logs just for testing.
I know about the file exporter but it's to verbose for the use case.

Thanks!

**Comment by bogdandrutu** [comment]
logging exporter exports to where you configure the application logger. What am I missing? Do you want just this https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/loggingexporter/logging_exporter.go#L71 ?
